### PR TITLE
Added health rest endpoint

### PR DESCRIPTION
**Pull request by jacobtomlinson**
Added `/healthz` endpoint which returns a `200` when you sent it a `GET`. This is a common health endpoint.

### CODE REVIEW

File: src/main/js/healthz.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+var winston = require("winston");
+
+module.exports = (req, res, next) => {
+    res.send('success');
+    winston.debug("Health endpoint called");
```
Comment (tpowellmeto) [inline src/main/js/healthz.js]: potentially a 'trace' message could clog up the logs a bit

Comment (jacobtomlinson) [inline src/main/js/healthz.js]: Winston doesn't appear to have a trace level. Plus this endpoint should only be hit in production where debug logging shouldn't be enabled.

Comment (tam203) [inline src/main/js/healthz.js]: I'm happy.

Comment (tam203) [inline src/main/js/healthz.js]: As discussed doesn't test much, maybe next step talk to luis, datapoint and gmaps.

**Pull request merged**